include/triton/Dialect/Triton/IR/Utility.h

```diff
@@ -25,7 +25,7 @@ template <typename Int> Int product(llvm::ArrayRef<Int> arr) {
 template <typename VecT> auto product(const VecT &vec) {
   return product(llvm::ArrayRef(vec));
 }
-template <typename Int> Int getNumElements(ArrayRef<Int> shape) {
+template <typename Int> Int getTotalSize(ArrayRef<Int> shape) {
```

Comment:
I think the old name was clearer.  The new name is ambiguous whether it's number of elements or number of bytes.

Reply:
`getNumElements` conflicts another MLIR internal function `getNumElements`...

Shall we instead move our `getNumElements` into the `triton` namespace?

Reply:
Sure, yes, I don't think we should define anything in the root MLIR namespace.